File: Questions-9th-July-2025/Python/001_question.py
def main():
    # TODO: Implement solution here
    n = int(input("Enter a number : "))
    while(n<0) : n = int(input("INVALID VALUE ENTERED!! \nEnter a positive value : "))
    if n%2==0 : print(n, "is even")
    else : print(n, "is odd")


if __name__ == "__main__":
    main()

File: Questions-9th-July-2025/C/001_question.c
#include <stdio.h>

int main() {
    int num;
    printf("Enter the number : ");
    do{
        scanf("%d", &num);
        if(num<0) printf("Negative value entered!! Try Again\n");
    }while(num<0);    
    printf((num%2) ? "Odd number" : "Even number");
    return 0;
}
